README.md

```diff
@@ -16,7 +16,7 @@ This is the official documentation platform for the [ReScript](https://rescript-
 
 ## System Requirements
 
-- `node@18` or higher
+- `node@20` or higher
```

Comment:
Core has bindings to `Array.toSorted`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted require node v20